packages/schematics/angular/ng-new/schema.json

```diff
@@ -95,19 +96,33 @@
     "routing": {
       "type": "boolean",
       "description": "Generates a routing module.",
-      "default": false
+      "default": false,
+      "x-prompt": "Will the project use Angular routing?"
     },
     "prefix": {
       "type": "string",
       "format": "html-selector",
       "description": "The prefix to apply to generated selectors.",
+      "minLength": 1,
       "default": "app",
-      "alias": "p"
+      "alias": "p",
+      "x-prompt": "Choose an HTML selector prefix for the project:"
```

Comment:
I don't think we should prompt for a selector. I think `app-` is fine for most folks. Super open to other ideas on this one.

Reply:
Removed.  Keeping it simple for now makes sense.  We can also add more later.